app/controllers/admin/content/system-images/subtopics.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+export default Controller.extend({
+  actions: {
+    openModal(subtopic) {
+      if (subtopic.name === 'Avatar') {
```

Comment:
But in template the modal is outside the loop. so there will be problem in checking.